ql/ql/src/queries/style/AndroidIdPrefix.ql

```diff
@@ -0,0 +1,32 @@
+/**
+ * @name Android query without android @id prefix
+ * @description Android queries should include the `android` prefix in their `@id`.
+ * @kind problem
+ * @problem.severity warning
+ * @id ql/android-id-prefix
+ * @precision high
+ */
+
+import ql
+
+string getIdProperty(QLDoc doc) {
+  result = any(string id | id = doc.getContents().splitAt("@") and id.matches("id %"))
+}
```

Comment:
Take a look at the `QueryDoc` class. 
That class has a utility predicate for getting the `@id` of a query.

Reply:
Love it, thanks! Applied in 4153480.